client/src/components/Header.js

```diff
@@ -43,6 +43,15 @@ const Header = () => {
               />
               JavaScript
             </NavDropdown.Item>
+            <NavDropdown.Divider />
+            <NavDropdown.Item as={Link} to="/node">
+              <FontAwesomeIcon
+                className="mr-2 text-warning"
+                aria-hidden="true"
+                icon={faJs}
```

Comment:
Can we use a Node icon?

Reply:
I will check.

Reply:
Yes it's a node icon so it fits our needs